crates/taplo/src/formatter/mod.rs

```diff
@@ -355,39 +358,62 @@ fn format_impl(node: SyntaxNode, options: Options, context: Context) -> String {
     formatted
 }
 
-struct FormattedEntry {
-    syntax: SyntaxElement,
-    key: String,
-    value: String,
-    comment: Option<String>,
+mod formatted_entry {
```

Comment:
Is there a reason for it to be separate module?

Reply:
The idea is to keep cleaned_key private, to make sure no one can initialize it any other way than using `replace().replace()` that was previously defined. Unless of course, you find it a bit overkill?

Reply:
You can keep all other fields public without making `cleaned_key` public, no?

Reply:
I'm not sure how to do that without the extra module layer. As it is right, now, it's already 100% private to this module (`formatter::mod.rs`). @ia0 seems to prefer a simple comment anyway, so I'm currently preparing this simplified version. Let me know what you think as well